File: AIRMAM5/Models/TSMapi/TapeInfoModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace AIRMAM5.Models.TSMapi
{
    /// <summary>
    /// 架上磁帶資訊 TapeInfoInLib
    /// <para> 參照 AIRMAM5.Tsm Models\\clsTSM.cs : clsTAPE_INFO_RESULT() </para>
    /// </summary>
    public class TapeInfoModel
    {
        /// <summary>
        /// 磁帶編號
        /// </summary>
        public string VOL_ID { get; set; } = string.Empty;

        /// <summary>
        /// 磁帶類型
        /// </summary>
        public string VOL_TYPE { get; set; } = string.Empty;

        /// <summary>
        /// 使用狀態
        /// </summary>
        public string VOL_USE_STATUS { get; set; } = string.Empty;

        /// <summary>
        /// 已存放資料(GB)
        /// </summary>
        public double USED_GB { get; set; }

        /// <summary>
        /// 最後讀取日期
        /// </summary>
        public string READ_DATE { get; set; } = string.Empty;

        /// <summary>
        /// 最後寫入日期
        /// </summary>
        public string WRITE_DATE { get; set; } = string.Empty;

        /// <summary>
        /// 儲存池
        /// </summary>
        public string POOL_NAME { get; set; } = string.Empty;

        /// <summary>
        /// 讀寫狀態
        /// </summary>
        public string VOL_RW_STATUS { get; set; } = string.Empty;

        /// <summary>
        /// 寫入錯誤次數
        /// </summary>
        public string WRITE_ERRORS { get; set; }

        /// <summary>
        /// 讀取錯誤次數
        /// </summary>
        public string READ_ERRORS { get; set; }
    }
}

File: AIRMAM5/Scripts/Models/Interface/Tsm/TapeInfoModel.ts
/** 架上磁帶資訊*/
export interface TapeInfoModel {
    /**磁帶編號 */
    VOL_ID: string;
    /** 磁帶類型 */
    VOL_TYPE: string;
    /**使用狀態 */
    VOL_USE_STATUS: string;
    /** 已存放資料(GB) */
    USED_GB: number;
    /** 最後讀取日期 */
    READ_DATE: string;
    /**最後寫入日期 */
    WRITE_DATE: string;
    /** 儲存池 */
    POOL_NAME: string;
    /** 讀寫狀態 */
    VOL_RW_STATUS: string;
    /**寫入錯誤次數 */
    WRITE_ERRORS: string;
    /**讀取錯誤次數 */
    READ_ERRORS: string;
}
